test/runtests.jl

```diff
@@ -2,12 +2,20 @@
 using FFTW
 using Base.Test
 
-# Make super sure we don't pull in Base names
 importall FFTW
-import AbstractFFTs: Plan, fft, ifft, bfft, fft!, ifft!, bfft!,
+if VERSION >= v"0.7.0-DEV.602"
+    import AbstractFFTs: Plan, fft, ifft, bfft, fft!, ifft!, bfft!,
```

Comment:
No, to avoid name conflicts, AbstractFFTs only exports things if none of the functions are defined in Base. So `using AbstractFFTs` won't import anything into Main until after the deprecation period.